.env.sample

```diff
@@ -0,0 +1,11 @@
+BASE_URL=""
+
+REPOSITORY_NAME=""
+USER_NAME=""
+USER_PASSWORD=""
+
+API_VERSION_PATH=""
+
+METADATA_NAME=""
+
+SESSION_LIMIT=10
```

Comment:
Do we need these environment variables in this repo?

Reply:
Currently these are required to send them to api
We should revisit options to authentication in order to give a better solution.